sycl/CMakeLists.txt

```diff
@@ -456,6 +456,7 @@ set( SYCL_TOOLCHAIN_DEPLOY_COMPONENTS
      sycl
      libsycldevice
      level-zero-sycl-dev
+     mlir-translate
```

Comment:
Does this mean it is also included in the `sycl-toolchain` build target?

Reply:
It was included already [see](https://github.com/intel/llvm/blob/cbbe6ddbaa1057f38d93476329b191504d252cc4/sycl/CMakeLists.txt#L359)